package.json

```diff
@@ -24,6 +24,7 @@
     "@reactioncommerce/file-collections-sa-gridfs": "0.0.1",
     "@reactioncommerce/job-queue": "^1.0.4",
     "@reactioncommerce/nodemailer": "^5.0.5",
+    "@reactioncommerce/hooks": "^1.0.1",
```

Comment:
I think for these core dependencies we should "hard pin" them to a specific version, so that a PR to core is still required before an upgrade happens.

Reply:
Discussed with Jeremy. I will do a separate PR to pin all of these